tests/code/test_events.py

```diff
@@ -511,6 +511,30 @@ def test_upcomingEvents():
                                 startDate = datetime.date(2021,12,12),
                                 endDate = datetime.date(2021,12,13))
 
+        newRecurringEvent = Event.create(name = "Recurring Event Test",
+                                term = 2,
+                                description = "Test upcoming program event.",
+                                location = "The sun",
+                                startDate = datetime.date(2021,12,12),
+                                endDate = datetime.date(2021,12,14),
+                                recurringId = 1)
+
+        newRecurringSecond = Event.create(name = "Recurring second event",
+                                term = 2,
+                                description = "Test upcoming program event.",
+                                location = "The sun",
+                                startDate = datetime.date(2021,12,13),
+                                endDate = datetime.date(2021,12,15),
+                                recurringId = 1)
+
+        newRecurringDifferentId = Event.create(name = "Recurring different Id",
+                                term = 2,
+                                description = "Test upcoming program event.",
+                                location = "The sun",
+                                startDate = datetime.date(2021,12,12),
+                                endDate = datetime.date(2021,12,13),
+                                recurringId = 2)
+
```

Comment:
getUpcomingEventsForUser orders events by start date so you want to make sure the start dates on the recurring events you create are correct so the test functions correctly.